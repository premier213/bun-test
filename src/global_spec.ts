import { expect, test, describe } from "bun:test";

describe("arithmetic", () => {
  test("2 + 2", () => {
    expect(2 + 2).toBe(4);
  });

  test("2 * 2", () => {
    expect(2 * 2).toBe(4);
  });
});

test("2 * 2", (done: () => void) => {
  Promise.resolve(2 * 2).then((result) => {
    expect(result).toEqual(4);
    done();
  });
});

test("snap", () => {
  expect("foo").toMatchSnapshot();
});
