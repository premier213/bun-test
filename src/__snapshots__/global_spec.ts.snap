// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`snap 1`] = `"foo"`;
